tracker/cli/update.py

```diff
@@ -0,0 +1,78 @@
+from click import echo, option, pass_context
+from . import cli
+
+
+@cli.group()
+def update():
+    """Update the application.
+
+    In doubt, call env as a general purpose command."""
+    pass
```

Comment:
Is this method really needed?

Reply:
yes this is a cli group, could be a lambda but the docstring is easier and more visible this way